pages/apis/rest_api/agents.md

```diff
@@ -6,8 +6,8 @@
 Returns a [paginated list](<%= paginated_resource_docs_url %>) of an organization's agents. The list only includes connected agents - agents in a disconnected state are not returned.
 
 ```bash
-curl -H "Authorization: Bearer $TOKEN" \
-  -X "https://api.buildkite.com/v2/organizations/{org.slug}/agents"
+curl -H "Authorization: Bearer $TOKEN" -X GET \
+  "https://api.buildkite.com/v2/organizations/{org.slug}/agents"
```

Comment:
Thanks very much for your PR @docwhat !

To match our other documentation, could I suggest that you associate the `-X GET` component of the command line with its value on the same line?

```suggestion
curl -H "Authorization: Bearer $TOKEN" \
  -X GET "https://api.buildkite.com/v2/organizations/{org.slug}/agents"
```